src/main.py

```diff
@@ -6,8 +6,10 @@
 import boto3
 import requests
 
+DEBUG = False
+
 codecommit = boto3.client("codecommit")
-slack_webhook = os.environ["SLACK_WEBHOOK"]
+slack_webhook = os.environ["SLACK_WEBHOOK"] if not DEBUG else None
```

Comment:
```suggestion
slack_webhook = os.environ.get("SLACK_WEBHOOK")
```

Reply:
DUH, don't know why I didn't think of this.